### PR TITLE
Calendar Label Reverted Back to Button

### PR DESCRIPTION
**Pull request by vazmymo**
We have fixed following items with this PR,
 - Changed Calendar Navigation from Label to Button
 - Replaced existing 5px padding with 10px top-padding

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,8 +24,7 @@ script:
 notifications:
   email:
     recipients:
-    - eajazali.mahaboobbasha@pearson.com
-    - veeramani.balagurusamy@pearson.com
+    - mohamed.azmy@pearson.com
```
Comment (umahaea) [inline .travis.yml]: could you reset this back to the original recipients? me and veera..

File: src/test/java/elementsTests/CalendarTest.java [hunk 2]
```diff
@@ -62,7 +62,8 @@ private void chooseEnv() {
 	}
```
Comment (umahaea) [inline src/test/java/elementsTests/CalendarTest.java]: code looks okay.. but the formatting is not good. Could you reformat the code. There are multiple line space for no reason.

**Comment by umahaea** [comment]
@vazmymo - could you rewrite this commit message following commitzen standards? follow one of our recent PR commit messages. __type_of_change(component_name): "actual code change"__

**Comment by vazmymo** [comment]
Closing the PR and will send the PR with the changes requested

**Pull request closed without merging**